### PR TITLE
feat: Spinner with label

### PR DESCRIPTION
**Pull request by ninariccimarie**

### CODE REVIEW

File: src/@next/Spinner/Spinner.tsx [hunk 1]
```diff
@@ -1,25 +1,39 @@
...
+  ['large']: 'subtitle1',
+};
 
-export const Spinner = (props: SpinnerProps) => (
-  <StyledWrapper className="spinner-container">
-    <Icon name="ri-loader" {...props} />
-  </StyledWrapper>
-);
+export const Spinner = ({ label, size, ...props }: SpinnerProps) => {
+  const [iconSize, setIconSize] = useState(null);
```
Comment (marthayaputra) [inline src/@next/Spinner/Spinner.tsx]: do we need this state? Since we don't change it locally

File: src/@next/Spinner/Spinner.tsx [hunk 2]
```diff
@@ -1,25 +1,39 @@
...
+
+  useEffect(() => {
```
Comment (marthayaputra) [inline src/@next/Spinner/Spinner.tsx]: Don't think we need an effect as well here

File: src/@next/Spinner/Spinner.tsx (hunk 1 above)
Comment (ninariccimarie) [reply src/@next/Spinner/Spinner.tsx]: you're right, let me remove them.

**Review by marthayaputra — APPROVED** [review]

**Review by chiahou — APPROVED** [review]
looks good to me. thanks

**Pull request merged**